README.md

```diff
@@ -137,6 +194,55 @@ export default async function handler(req, res) {
 
   return res.status(200).json({ message: 'ok' });
 }
+```
+#### API - App Router Example
+
+`/app/api/hello/route.ts`
+
+```ts
+import type { NextRequest, NextResponse } from 'next/server';
+import { deleteCookie, getCookie, setCookie, hasCookie, getCookies } from 'cookies-next';
+
+export async function GET(req: NextRequest) {
+  const res = new NextResponse();
+  setCookie('test', 'value', { res, req });
+  getCookie('test', { res, req });
+  getCookies({ res, req });
+  deleteCookie('test', { res, req });
+  hasCookie('test', { req, res });
+
+  // provide cookies fn
+  setCookie('test1', 'value', { cookies });
```

Comment:
please add where we can import "cookies" from

Reply:
He imported from next headers 

```
import { cookies } from 'next/headers';
```

Reply:
Yes, I know 
But it will be better if it's in the documentation

Reply:
Fixed [af69b41](https://github.com/andreizanik/cookies-next/pull/55/commits/af69b4172afe3e7602b7fa11a98b622610a6572e)
Thanks for review 🙏